models/searchQuery.ts

```diff
@@ -0,0 +1 @@
+export const fields: string[] = ['query', 'amount']
```

Comment:
The `: string[]` annotation is unnecessary in this case because the type is [inferred from the value](https://www.typescriptlang.org/docs/handbook/type-inference.html).